pom.xml

```diff
@@ -12,8 +12,10 @@
 
     <modules>
         <module>pizza-order-showcase</module>
+        <module>restaurant-showcase</module>
         <module>process-example</module>
         <module>schema-client-example</module>
         <module>customer-onboarding-showcase</module>
+        <module>restaurant-showcase/satisfaction-miranum</module>
```

Comment:
Why this?

Reply:
maven placed it there after creating a new module. I think that can be deleted